usaon_vta_survey/util/db/connect.py

```diff
@@ -31,8 +31,9 @@ def db_connstr(app: Flask) -> str:
         return connstr
     else:
         # TODOL figure out why os.environ doesn't work here
-        host = os.getenv('USAON_VTA_DB_HOST')
-        port = os.getenv('USAON_VTA_DB_PORT')
+        host = os.environ['USAON_VTA_DB_HOST']
+        # host = os.getenv('USAON_VTA_DB_HOST')
+        port = os.environ['USAON_VTA_DB_PORT']
         user = os.getenv('USAON_VTA_DB_USER')
         password = os.getenv('USAON_VTA_DB_PASSWORD')
```

Comment:
@rmarow let's not mix `os.environ` and `os.getenv` :) There is actually a difference - `os.getenv` will return `None` by default if an environment variable isn't found, and I think we want to throw an error instead.